web/webdashboard/footer_.html

```diff
@@ -15,6 +15,19 @@
     }
 </script>
 
+<script type="module">
+    import { JSONEditor } from '/static/vanilla-jsoneditor.js'
+
+    document.querySelectorAll('.jsoneditor').forEach((element) => {
+        let content = { json: JSON.parse(element.innerText), text: undefined }
+        element.innerHTML = ''
```

Comment:
Reminder, not affecting this PR: https://linear.app/autokitteh/issue/ENG-788/solve-xss-issue-with-dashboard-messages